src/User32.Desktop/User32.Helpers.cs

```diff
@@ -47,5 +47,24 @@ public static unsafe string GetClassName(IntPtr hWnd)
 
             return new string(className, 0, count);
         }
+
+        /// <summary>
+        /// Retrieves from the clipboard the name of the specified registered format.
+        /// </summary>
+        /// <param name = "format">The type of format to be retrieved. This parameter must not specify any of the predefined clipboard formats.</param>
+        /// <returns>The format name string.</returns>
+        /// <exception cref="Win32Exception">Thrown when an error occurs.</exception>
+        public static unsafe string GetClipboardFormatName(int format)
+        {
+            const int bufferSize = 256;
```

Comment:
I believe you meant "there is _no_ way".
Given that, the Helper function probably shouldn't just guess and limit the caller. @arlm can you add a `maxLength` parameter to this helper method, with a default argument (of 256 if you want) so that the caller doesn't have to, but *may* override this value if they choose?
Although I suppose if it can be a user-supplied value then it's not safe to use `stackalloc` since it may be a very large value. In fact, C# may not even let you pass in a non-constant value to a `stackalloc` use. So you may need to allocate the array on the GC heap instead.
What do you think? Is 256 bytes long enough for any clipboard format?